sdk/core/core-client/review/core-client.api.md

```diff
@@ -5,6 +5,8 @@
 ```ts
 
 import { AbortSignalLike } from '@azure/abort-controller';
+import { createSpanFunction } from '@azure/core-tracing';
+import { CreateSpanFunctionArgs } from '@azure/core-tracing';
```

Comment:
is this worth having a named type for? I am leaning towards saying we should inline it just to reduce surface further

Reply:
Yeah, good call. Removed it. But I'm keeping `SpanConfig`! 😆